test/integration/banListTest.ts

```diff
@@ -239,7 +238,7 @@ describe('Test: ACL updates will batch when rules are added in succession.', fun
 
         // Setup some protected rooms so we can check their ACL state later.
         const protectedRooms: string[] = [];
-        for (let i = 0; i < 10; i++) {
+        for (let i = 0; i < 5; i++) {
```

Comment:
what's the rationale behind this being `5`?

Reply:
because it keeps failing in the CI for taking so long. The number of protected rooms takes time because as we change the server rules in the watched list, we emit a tonne of `m.room.server_acl` events in each protected room. Crucially though `ApplyServerAcls` in Mjolnir fetches the state event (to see if it is the same) and then sets it and both of those requests take a large proportion of time  

![image](https://user-images.githubusercontent.com/50846879/184872318-7b3c1de6-b96f-4f31-a3d6-2774c1a06b2a.png)

Though i can't say i'm using this profiler properly, i was just playing around with it yesterday. You don't need to use chromium btw there's also an extension for it in VSCode https://github.com/microsoft/vscode-js-profile-visualizer